rootfs/etc/nginx/lua/global_throttle.lua

```diff
@@ -62,11 +62,13 @@ local function get_namespaced_key_value(namespace, key_value)
   return namespace .. key_value
 end
 
-local function set_headers(limit, count_current, window_size)
-      ngx.header["Ratelimit-Remaining"] = string.format("%.0f",
-        (limit - count_current <= 0 ) and 0 or (limit - count_current))
-      ngx.header["Ratelimit-Limit"] = limit
-      ngx.header["Ratelimit-Window"] = window_size
+local function set_headers(limit, estimated_final_count, window_size, remaining_time)
+  ngx.header["Ratelimit-Remaining"] = estimated_final_count == nil and {}
```

Comment:
In the event either estimated_final_count or remaining_time are nil, asociated headers will not be set.